README.md

```diff
@@ -326,6 +326,9 @@ Table of Contents
 ### • Video Player
 - [**VLC**](https://www.videolan.org/vlc/download-android.html) <sup>**[[F-Droid](https://f-droid.org/app/org.videolan.vlc)]**</sup>
 - [**mpv for Android**](https://github.com/mpv-android/mpv-android)
+- [**Nova Video Player**](https://github.com/nova-video-player/aos-AVP) <sup>**[[F-Droid](https://f-droid.org/app/org.courville.nova)]**</sup>
+- [**Fermata Media Player**](https://github.com/AndreyPavlenko/Fermata)
+- [**Just (Video) Player**](https://github.com/moneytoo/Player) <sup>**[[F-Droid](https://f-droid.org/packages/com.brouken.player)]**</sup>
```

Comment:
Just a minor correction for consistency: Please use the `app` url instead of `packages`.

Reply:
When I use `app` instead of `packages`, F-Droid site throws 404. So I triead to be as much consistent as possible with your other links.

Reply:
The trick is: The `app` url must not contain a trailing `/`.

https://f-droid.org/app/com.brouken.player